src/fiatExchanges/SelectProvider.tsx

```diff
@@ -129,7 +130,10 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
 
   const asyncExchanges = useAsync(async () => {
     try {
-      const availableExchanges = await fetchExchanges(userLocation.countryCodeAlpha2, digitalAsset)
+      const availableExchanges = await fetchExchanges(
+        userLocation.countryCodeAlpha2,
+        tokenInfo.symbol
```

Comment:
does this also need to be upper cased?

Reply:
I don't think so, before it was just passing in `digitalAsset` which was equivalent to the token symbol. It is uppercased below for fetchProviders because there it was changing it to uppercase before through an enum map.